examples/dt/uni07eta/control-plane/service-values.yaml

```diff
@@ -58,7 +58,7 @@ data:
     glanceAPIs:
       default:
         preserveJobs: false
-        replicas: 1
+        replicas: 3
         type: split
     storageClass: ""
     storageRequest: 1G
```

Comment:
@fultonj should this need to be `10` instead of `1` ? or did we have `1` on purpose?

Reply:
We can use 10. I don't believe 1 was used for any specific reason.